### DISCUSSION TITLE
Changing default font color from gray text

### DISCUSSION THREAD

**Discussion by ordo-n** [jcubic/jquery.terminal#960] [Q&A] [answered]
How do I change the default font color from gray to white?

I'm struggling to get it working, I have tried multiple attempts with css and options

i dont want it to effect the html contents in a raw echo either

**Comment by ordo-n** [comment, marked as answer]
turns out it needs the double dash --

 .terminal {
        --color: #ffffff;
    }

**Reply by jcubic** [reply]
Yes, they are called [Custom Properties or CSS variables](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties) and this is the way to [customize the style of the Terminal](https://github.com/jcubic/jquery.terminal/wiki/CSS-Style).
